### PR TITLE
Add Mohamed Omar and Sunil Ravipati as 2023H2 SSC election candidates 

### PR DESCRIPTION
**Pull request by dfeldman**

### CODE REVIEW

**Review by amoore877 — DISMISSED** [review]

File: ssc/elections/2023H2/MOHAMED_OMAR.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Mohamed Omar
+Sunil Ravipati leads SPIFFE/SPIRE efforts at Confluent.
```
Comment (azdagron) [inline ssc/elections/2023H2/MOHAMED_OMAR.md]:
```suggestion
Mohamed Omar leads SPIFFE/SPIRE efforts at Confluent.
```

**Review by amoore877 — APPROVED** [review]

**Review by v0lkan — APPROVED** [review]
🌮

**Comment by v0lkan** [comment]
That taco ought to have been a thumbs up; yet 1. I like it :) — 2. I’ll just roll with it. 

3. you may see more tacos from me :D .

File: ssc/elections/2023H2/MOHAMED_OMAR.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Mohamed Omar
+Mohamed Omar leads SPIFFE/SPIRE efforts at Confluent.
+He has participated in the SPIFFE community for over 2 years and would be an excellent member of the steering committee.
+
+## Reference
+**Email:** momar@confluent.io
+**GitHub:** https://github.com/moemar
```
Comment (rturner3) [inline ssc/elections/2023H2/MOHAMED_OMAR.md:7]: I believe it should be https://github.com/moe-omar, this is pointing to a different person on GitHub.

**Pull request merged**